src/test/java/core/basesyntax/database/StorageDaoImplTest.java

```diff
@@ -0,0 +1,80 @@
+package core.basesyntax.database;
+
+import core.basesyntax.model.Fruit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageDaoImplTest {
+    private static StorageDaoImpl storageDao;
+    private static List<Fruit> fruitList;
+    private Fruit fruit;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        storageDao = new StorageDaoImpl();
+        fruitList = new ArrayList<>();
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        fruit = new Fruit("apple");
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        Storage.fruits.clear();
+    }
+
+    @Test
+    public void addFruit_ok() {
+        storageDao.add(fruit);
+        boolean actual = Storage.fruits.isEmpty();
+        Assert.assertFalse("Fruit hasn't added to storage", actual);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void addExistingFruit_notOk() {
+        storageDao.add(fruit);
```

Comment:
```suggestion
        Storage.fruits.add(fruit);
```
Ты ожидаешь ошибку при вызове второго метода, а этой строчкой ты только создаёшь необходимые условия для тестирования, поэтому лучше обращаться к storage напрямую